maven-plugin/src/main/java/org/openmrs/maven/plugins/model/ModuleProperties.java

```diff
@@ -0,0 +1,206 @@
+package org.openmrs.maven.plugins.model;
+
+import org.apache.commons.lang.StringUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+
+/**
+ *
+ */
+public class ModuleProperties {
```

Comment:
This name is a bit misleading, maybe we could name it "BaseSdkProperties". I guess this class could be abstract as well